components/registry/org.wso2.carbon.registry.resource.ui/src/main/resources/web/resources/xml_resource_visualizer_ajaxprocessor.jsp

```diff
@@ -16,8 +16,9 @@
  ~ under the License.
  --%>
 <%@ page contentType="application/xml;charset=UTF-8" language="java" pageEncoding="UTF-8" %>
+<%@ page import="org.owasp.encoder.Encode" %>
 <?xml-stylesheet type="text/xsl"
-    href="../resources/xslt/annotated-<%=request.getParameter("type")%>.xsl"?>
+    href="../resources/xslt/annotated-<%=Encode.forHtmlAttribute(request.getParameter("type"))%>.xsl"?>
```

Comment:
SInce here the content type is xml, shouldn't we be using forXmlAttribute function instead?

Reply:
forHtmlAttribute internally use forXmlAttribute. Both are effectively equal. +1 to go ahead with that we already merged. https://github.com/OWASP/owasp-java-encoder/blob/9645e8dad1c436dae98382a28549ff8d745ae3c0/core/src/main/java/org/owasp/encoder/Encode.java#L308